packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -176,6 +176,10 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     let curr = this.refs.root;
     while (window.getComputedStyle(curr).getPropertyValue('background-color') === 'rgba(0, 0, 0, 0)' ||
       window.getComputedStyle(curr).getPropertyValue('background-color') === 'transparent') {
+      if (curr.tagName === 'HTML') {
+        // Fallback color if no element has a declared background-color attribute
+        return 'rgba(244, 244, 244, 1)';
```

Comment:
We should just use `return;` instead of returning a hard-coded value.

Reply:
It does need some fallback value since without one it defaults to transparent:

![transparent-sticky](https://user-images.githubusercontent.com/3977969/35542678-177e047a-0516-11e8-87cd-242f2b8577e8.png)

I'm updating to assign it outside the function. Let me know your thoughts on the updated solution.